libplatsch.c

```diff
@@ -166,6 +169,33 @@ static void platsch_custom_draw_buffer(struct platsch_ctx *ctx,
 	ctx->custom_draw_buffer_cb(&buf, ctx->custom_draw_priv);
 }
 
+static void drmconnector_add(struct modeset_dev *dev, uint32_t connector_id)
+{
+	unsigned int i;
+
+	for (i = 0; i < dev->max_conn_num; i++) {
+		if (dev->conn_id[i] == 0)
+			break;
+	}
```

Comment:
Why is this iteration needed? AFAICS you already know the last used array entry by looking at `dev->conn_num`.